pyvrp/cpp/CostEvaluator.h

```diff
@@ -132,7 +139,6 @@ class CostEvaluator
      */
     template <bool exact = false,
               bool skipLoad = false,
-              bool skipDuration = false,
```

Comment:
We don't actually use this anywhere, so for now let's remove it. We might add it back in once we have a suitable use case.